elementary/monitor/cli.py

```diff
@@ -166,15 +166,22 @@ def monitor(
     '--disable-passed-test-metrics',
     type=bool,
     default=False,
-    help="If set to true elementary report won't show data metrics for passed tests (this can improve report creation time)."
+    help="If set to true elementary report won't show data metrics for passed tests (this can improve report "
+         "creation time)."
+)
+@click.option(
```

Comment:
We have the `common_options` function above that is designed to aggregate CLI options that are received on all the `monitor` commands.

Could you please move the `click.option` to there?

Reply:
At the beginning I placed it in the common_options but then I realized that its not really used in the "monitor" command so I thought it might look confusing and changed. 
No prob, moving to common_options